dotnet/src/webdriver/Command.cs

```diff
@@ -31,16 +31,6 @@ public class Command
         private string commandName;
         private Dictionary<string, object> commandParameters = new Dictionary<string, object>();
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="Command"/> class using a command name and a JSON-encoded string for the parameters.
-        /// </summary>
-        /// <param name="name">Name of the command</param>
-        /// <param name="jsonParameters">Parameters for the command as a JSON-encoded string.</param>
-        public Command(string name, string jsonParameters)
```

Comment:
Double check why it is removed.

Reply:
It was removed due to it was not used, returned it back as publicly shipped api.